Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1488,7 +1488,9 @@ def return_outputs(readable_output, outputs, raw_response=None):
         "Contents": raw_response,
         "EntryContext": outputs
     }
-
+    # Return 'ok' to test module
+    if readable_output == 'ok' and not outputs and not raw_response:
+        demisto.results('ok')
```

Comment:
when we return in test any string other than `ok` then the test should fail and present that error. Please check that scenario